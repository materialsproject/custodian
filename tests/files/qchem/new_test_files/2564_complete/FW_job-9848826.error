/global/u2/s/sblau/pymatgen/pymatgen/io/cif.py:44: UserWarning: Please install optional dependency pybtex if youwant to extract references from CIF files.
  warnings.warn("Please install optional dependency pybtex if you"
DEBUG:launchpad:FW with id: 2564 is unique!
DEBUG:launchpad:Created/updated Launch with launch_id: 4102
DEBUG:launchpad:RUNNING FW with id: 2564
INFO:rocket.launcher:RUNNING fw_id: 2564 in directory: /scratch2/scratchdirs/sblau/fragment_wf/block_2018-07-09-23-50-25-765742/launcher_2018-07-12-01-21-14-704179
INFO:rocket.launcher:Task started: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}}.
INFO:rocket.launcher:Task completed: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}} 
INFO:rocket.launcher:Task started: {{atomate.qchem.firetasks.run_calc.RunQChemCustodian}}.
INFO:custodian.custodian:Run started at 2018-07-11 18:22:11.194920 in /scratch2/scratchdirs/sblau/fragment_wf/block_2018-07-09-23-50-25-765742/launcher_2018-07-12-01-21-14-704179.
INFO:custodian.custodian:Custodian running on Python version 3.6.4 |Anaconda, Inc.| (default, Jan 16 2018, 18:10:19)  [GCC 7.2.0]
INFO:custodian.custodian:Hostname: nid04298, Cluster: unknown
INFO:custodian.custodian:Starting job no. 1 (QCJob) attempt no. 1. Total errors and errors in job thus far = 0, 0.
srun: error: nid04298: task 0: Aborted
srun: Terminating job step 9848826.0
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:root:Backing up run to error.1.tar.gz.
ERROR:custodian.custodian:
{ 'actions': [ { 'molecule': 'molecule_from_last_geometry'}],
  'errors': [ 'out_of_opt_cycles'],
  'handler': <custodian.qchem.handlers.QChemErrorHandler object at 0x2aaabb42e5f8>}
INFO:custodian.custodian:Starting job no. 1 (QCJob) attempt no. 2. Total errors and errors in job thus far = 1, 1.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Starting job no. 2 (QCJob) attempt no. 1. Total errors and errors in job thus far = 1, 0.
INFO:custodian.custodian:QCJob.run has completed. Checking remaining handlers
INFO:custodian.custodian:Logging to custodian.json...
INFO:custodian.custodian:Run ended at 2018-07-11 19:11:17.358204.
INFO:custodian.custodian:Run completed. Total time taken = 0:49:06.163284.
