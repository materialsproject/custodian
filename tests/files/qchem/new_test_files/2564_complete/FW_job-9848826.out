foobar
2018-07-11 18:22:02,681 INFO Hostname/IP lookup (this will take a few seconds)
2018-07-11 18:22:02,684 INFO Launching Rocket
2018-07-11 18:22:11,159 INFO RUNNING fw_id: 2564 in directory: /scratch2/scratchdirs/sblau/fragment_wf/block_2018-07-09-23-50-25-765742/launcher_2018-07-12-01-21-14-704179
2018-07-11 18:22:11,183 INFO Task started: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}}.
2018-07-11 18:22:11,191 INFO Task completed: {{atomate.qchem.firetasks.write_inputs.WriteInputFromIOSet}} 
2018-07-11 18:22:11,194 INFO Task started: {{atomate.qchem.firetasks.run_calc.RunQChemCustodian}}.
